packages/nextjs/pages/projects/[projectId].tsx

```diff
@@ -1,88 +1,147 @@
 import React, { useEffect, useState } from "react";
-import type { NextPage } from "next";
+import { NextPage } from "next";
 import { GetServerSideProps } from "next";
-import { HeartIcon as HeartFilledIcon } from "@heroicons/react/20/solid";
-import { HeartIcon } from "@heroicons/react/24/outline";
-import { Address } from "~~/components/scaffold-eth";
-import SuggestProjects from "~~/components/shared/SuggestProjects";
-import { useBallot } from "~~/context/BallotContext";
+import { AiOutlineGithub } from "react-icons/ai";
+import { BsGlobe } from "react-icons/bs";
+import { IoMdOpen } from "react-icons/io";
+import { useAccount } from "wagmi";
+import { ArrowTopRightOnSquareIcon } from "@heroicons/react/20/solid";
+import YourBallot from "~~/components/op/projects/YourBallot";
+import IncludedProjects from "~~/components/project/IncludedProjects";
+import ProjectHeader from "~~/components/project/ProjectHeader";
+import Sidebar from "~~/components/shared/Sidebar";
 import dbConnect from "~~/lib/dbConnect";
 import Project, { ProjectDocument } from "~~/models/Project";
-import { notification } from "~~/utils/scaffold-eth";
 
 interface Props {
   projects: ProjectDocument[];
 }
 
 const ProjectDetail: NextPage<Props> = ({ projects }) => {
-  const [isLiked, setIsLiked] = React.useState(false);
-  const [isAdded, setIsAdded] = useState(false);
-  const { state, dispatch } = useBallot();
+  const { isDisconnected } = useAccount();
+  const [wallet, setWallet] = useState<boolean | false>(false);
```

Comment:
Why do you need the "| false"?

Reply:
Most of my PR was just copying and pasting the existing UI pages that we had built prior to the database addition. So there were a few things like this that I have no idea why they existed but I have corrected them. Good catch.